models/inceptionv3/mybuild-tizen.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+## @brief The simple build script for Tizen 6.0 platform
+#  @Author Geunsik Lim <geunsik.lim@samsung.com>
+#
+#  @note
+#   Build source code on Tizen 6.0 ARM 32bit on VIM3 board (Vivante)
+#   - tizen# zypper install binutils gcc libgcc gcc-c++ cpp
+#   - tizen# zypper install amlogic-vsi-npu-sdk-devel
+#   - tizen# zypper install dann-devel
+#   - tizen# zypper install acuity-ovxlib-devel
+#   - tizen# ./mybuild-tizen.sh
+
+
+# ---------------------- Initialization area -------------------------------------
+rm -rf ./bin_r/
+mkdir  ./bin_r/
+
+# ---------------------- configuration area -------------------------------------
+shared=1
+
+# ---------------------- build area -----------------------------------------------
+# Handle one between Ubuntu/ARM and Tizen/ARM
+export header_dir="/usr/include"
+export compiler="gcc -g"
+export compile_flag=""
+
+# Handle one between dynaming linking and static linking (ELF).
+if [[ $shared -eq 1 ]]; then
+    export pic_set="-fPIC"
+    export shared_set="-shared"
+else
+    export pic_set=" "
+    export shared_set=" "
+fi
+
+export conf_option=" "
+export my_header="-I/usr/include/dann -I/usr/include/ovx -I/usr/include/amlogic-vsi-npu-sdk "
```

Comment:
`dann` is required?

Reply:
It's optional. Let's remove the content because we will append RPM packages with Vivante NPU Toolkit Libraries (GPLv2) in the near future. Updated.